apps/confidential/confidential.go

```diff
@@ -418,6 +418,12 @@ func (cca Client) AcquireTokenOnBehalfOf(ctx context.Context, userAssertion stri
 	return cca.base.AcquireTokenOnBehalfOf(ctx, params)
 }
 
+// Accounts gets all the accounts in the token cache.
+// If there are no accounts in the cache the returned slice is empty.
+func (cca Client) Accounts() []Account {
```

Comment:
No, in confidential client we'd like to avoid `GetAccounts` operation. Instead, use `GetAccount(id)`. 
The reason for this is that `GetAccounts()` does not play well with distributed caching. 

In both web site and web api scenario, the token cache cannot be stored in memory or in a file, since its size is proportional to the number of users.